pkg/storkctl/action.go

```diff
@@ -86,3 +131,33 @@ func anyActionIncomplete(namespace string) bool {
 func newActionName(action storkv1.ActionType) string {
 	return strings.Join([]string{string(action), time.Now().Format(nameTimeSuffixFormat)}, "-")
 }
+
+func waitForActionToComplete(actionName, namespace string) (bool, error) {
+	action, err := task.DoRetryWithTimeout(
+		func() (interface{}, bool, error) {
+			action, err := storkops.Instance().GetAction(actionName, namespace)
+			if err != nil {
+				return nil, true, err
+			}
+			return action, isActionIncomplete(action), nil
+		},
+		actionWaitTimeout,
+		actionWaitInterval)
+	if err != nil {
+		return false, err
+	}
+	if action.(*storkv1.Action).Status == storkv1.ActionStatusFailed {
+		return false, nil
+	}
+	return true, nil
+}
+
+func getDebugMessage(actionName, namespace string) string {
+	return "debug info:\n" +
+		"- for details on the action, use: " + getCmdDescribeAction(actionName, namespace) +
+		"\n- to view stork logs, use: kubectl -n kube-system logs -l name=stork"
```

Comment:
Stork could be running in any other namespace apart from `kube-system`. We can make this generic 
```
kubectl -n <stork-namespace> logs -lname=stork
```